Dockerfile

```diff
@@ -70,7 +73,7 @@ FROM base AS build
 ARG TARGETPLATFORM
 ARG QEMU_VERSION QEMU_TARGETS
 ENV AR=llvm-ar STRIP=llvm-strip
-RUN --mount=target=.,from=src,src=/src/qemu,rw --mount=target=./install-scripts,src=scripts \
+RUN --mount=target=.,from=src-clone,src=/src/qemu,rw --mount=target=./install-scripts,src=scripts \
```

Comment:
`from=src`

Reply:
`src` has an unnecessary COPY for this case compared to `src-clone`

Reply:
Or does buildkit optimize this out and skips the copy ?

Reply:
but it ensures that there are no 2 sources of truth